### PR TITLE
added checkbox interactive and added localStorage

### PR DESCRIPTION
**Pull request by AtillaTahak**

### CODE REVIEW

File: src/js/checkbox.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+export const updateLocal = (getList) => {
+  localStorage.clear();
+  localStorage.setItem('to-do', JSON.stringify(getList));
+};
+
+export function checkList(check, listChek) {
```
Comment (rloterh) [inline src/js/checkbox.js:6]: - [ ] Completed **tasks are not updating correctly in the local storage**. When _task one_ is checked as completed, the _task two_ status is rather updated in the local storage. Kindly ensure that the appropriate task's completed status is updated in the local storage when that particular task is checked

**Your app (on taschange status):**
image
s):

**Review by rloterh — APPROVED** [review]
Hi Atilla, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Module import correctly implemented
- No linter errors

## Required Changes ♻️
_Kindly check the comments under the review._


<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._## Status: Approved :heavy_check_mark:

File: src/js/checkbox.js (hunk 1 above)
Comment (AtillaTahak) [reply src/js/checkbox.js:6]: thank you I didn't realize. But now I solved

Comment (henatan99) [reply src/js/checkbox.js:6]: - [ ] @AtillaTahak , I appreciate the effort you made to fix this issue. Unfortunately, the issue persists. I think you need to test your `checkList` function as I suspect the issue is originated from there. Kindly make changes and also make sure the app works as intended before submitting for another code review request.

Screenshot from 2021-11-04 02-19-46

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @AtillaTahak , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
# Hi @AtillaTahak 👋, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**